pkg/imake

```diff
@@ -0,0 +1,23 @@
+[mirrors]
+https://www.x.org/releases/individual/util/imake-1.0.7.tar.gz
+
+[vars]
+filesize=195387
+sha512=d45bd491bcd2a0490d3a622b5f4691929e32451a2dff7ab96f75fe0df84a94375a8932dd495da680ce5bafa6b9b71bb8a7b9ee6e44b88b20c2c462678c450d79
+pkgver=1
+desc="Utility for creating/using imake files - from xorg-utils"
+
+[deps]
```

Comment:
can you make that [deps.host] please ? pkg-config is a tool running on the host (which is an important distinction for crosscompiles).
also please use "pkgconf" instead of "pkg-config" as all the other packages do